website/docs/terms/idempotent.md

```diff
@@ -0,0 +1,19 @@
+---
+id: idempotent
+title: Idempotent
+displayText: idempotent
+hoverSnippet: Idempotent describes a process that is independent of previous executions of that same process.
+---
+:::important This page needs some love
```

Comment:
Couple things with this CTA:

1) Would soften "needs" to "could use" - needs implies that the definition that's there is incorrect
2) The GH repo isn't "ours", it's just the repo for docs
3) "Requires more content" could be more specific on the type of content it could use

Putting these together, something like:

:::important This page could use some love
This term would benefit from additional depth and examples.  Have knowledge to contribute? [Create a discussion in the docs.getdbt.com GitHub repository](https://github.com/dbt-labs/docs.getdbt.com/discussions) to begin the process of becoming a glossary contributor!
:::

Reply:
@krevitt @johnblust  I agree this message could use some love! Some thoughts:
* Can we make this message a Snippet so it can be reused on all pages that need love?
* Can we use a more universal phrasing for non-native English speakers? It might actually translate to some unfortunate things in google translate. Maybe we could title it something like, "Opportunity for improvement" with your message above.

Reply:
Good callout! I fixed this in the latest commit.